Library/Homebrew/cmd/search.rb

```diff
@@ -157,7 +157,7 @@ def search_formulae(rx)
       if aliases.include?(name) && results.include?(canonical_full_name)
         next
       elsif (HOMEBREW_CELLAR/canonical_name).directory?
-        "#{name} (installed)"
+        pretty_installed(name)
```

Comment:
Sorry to be this blunt, but this change completely ignores the way the result of `search_formulae` is used by the caller:

1. It relies on the `(installed)` suffix to be there to know which items to highlight in the output.
2. The list is passed to `puts_columns` for multi-column output and this function can only deal with plain text without any embedded formatting (but `pretty_installed` adds formatting). That's why `puts_columns` has a second argument to list the items that should be highlighted and does the highlighting internally. Unless `puts_columns` is extended to understand a bare minimum of ANSI escape codes, it won't be able to create a proper multi-column layout with this kind of input.